### PR TITLE
Sign transactions in CLI

### PR DESCRIPTION
**Pull request by DieMyst**

### CODE REVIEW

File: cli/src/contract_func.rs [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    where
+        P: Tokenize,
+    {
+        if let Some(p) = password {
+            web3.personal().unlock_account(account, p, None).wait()?;
+        }
+
+        let contract = utils::init_contract(&web3, self.contract_address)?;
+
+        let result_code_publish = contract.call(func, params, account, options);
```
Comment (alari) [inline cli/src/contract_func.rs]: Why `code_publish` is there?

File: cli/src/contract_func.rs [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        P: Tokenize,
+        R: Detokenize,
+    {
+        let (_eloop, transport) = web3::transports::Http::new(&self.eth_url)?;
+        let web3 = web3::Web3::new(transport);
+
+        let contract = utils::init_contract(&web3, self.contract_address)?;
+
+        let result_code_publish = contract.query(func, params, None, utils::options(), None);
+        let res = result_code_publish.wait()?;
```
Comment (alari) [inline cli/src/contract_func.rs]: It looks like `result` is enough, isn't it?

File: cli/src/contract_func.rs (hunk 1 above)
Comment (DieMyst) [reply cli/src/contract_func.rs]: copy-paste, will change it

File: cli/src/contract_func.rs (hunk 2 above)
Comment (DieMyst) [reply cli/src/contract_func.rs]: yes

File: cli/src/register.rs [hunk 3]
```diff
@@ -102,35 +113,58 @@ impl Register {
...
+                &self.credentials,
                 "addNode",
                 (
                     self.tendermint_key,
                     hash_addr,
                     u64::from(self.min_port),
                     u64::from(self.max_port),
                 ),
-                options,
+                1_000_000,
```
Comment (alari) [inline cli/src/register.rs]: Why this magic number, why not some another one?

File: tools/run/cleanup.sh [hunk 4]
```diff
@@ -2,11 +2,11 @@
 
 case "$(uname -s)" in
    Darwin)
-     docker-compose -f macos/docker-compose.yml kill
+     docker-compose -f docker-compose.yml kill
      ;;
 
    Linux)
-     docker-compose -f linux/docker-compose.yml kill
+     docker-compose -f docker-compose.yml kill
```
Comment (alari) [inline tools/run/cleanup.sh]: Is there any reason to keep the switch cases?

**Pull request merged**